### ISSUE TITLE
All node examples on try.scroll.pub

### ISSUE THREAD

**Issue by alexjj** [breck7/tryscroll#1]
Hello - I've been curious about scroll for some time. It's not something I've seen and am keen to learn about it. The addition of https://try.scroll.pub has been great to understand the syntax and quickly see how things work.

It would be great if all node types are shown on the page - as defined [here](https://github.com/publicdomaincompany/scroll/blob/main/scrolldown.grammar).

**Comment by breck7** [comment]
> The addition of https://try.scroll.pub has been great to understand the syntax and quickly see how things work.

Glad to hear you are finding it helpful.

> It would be great if all node types are shown on the page - as defined here, and maybe capturing items from the release notes.

This is a good idea. The TryScrollApp also supports loading from URLs. So I can open the ReleaseNotes in one link: https://try.scroll.pub/#url%20https://scroll.pub/releaseNotes.scroll

Perhaps a "demo.scroll" or "reference.scroll" file could be created with all of the default node types shown.

My main concern right now is to try and come up with some more interesting node types. Are there any types that you'd like to see?

**Comment by alexjj** [comment]
> The TryScrollApp also supports loading from URLs. So I can open the ReleaseNotes in one link: https://try.scroll.pub/#url%20https://scroll.pub/releaseNotes.scroll
Oh that's neat! 

> My main concern right now is to try and come up with some more interesting node types. Are there any types that you'd like to see?

I liked the issue you raised about id on nodes. Being able to put `paragraph #my-id` would be neat. Bit like how textile does it. It could add a # or some symbol that's an anchor to it, so people can click it or copy the link.

If I were making requests, it would be to adjust the style of the index page. I'm not sure where I put custom css for the index page?

**Comment by breck7** [comment]
> It could add a # or some symbol that's an anchor to it, so people can click it or copy the link.

I agree tagging paragraphs with ids is helpful. Some sugar like `paragraph #my-id` would probably be nice. For now I just added the naive way of adding an "id " line.

```
aftertext
 Click here to go to the middle paragraph.
 link #middleParagraph Click here

loremIpsum 20

aftertext
 Here is the middle paragraph.
 id middleParagraph

loremIpsum 20
```
